translations/ru.po

```diff
@@ -3,25 +3,25 @@ msgid ""
 msgstr ""
 "Project-Id-Version: GodSVG\n"
 "POT-Creation-Date: \n"
-"PO-Revision-Date: \n"
-"Last-Translator: volkov <volkovissocool@gmail.com>\n"
-"Language-Team: \n"
+"PO-Revision-Date: 2024-05-31 11:16+0300\n"
+"Last-Translator: Gallifreyan <gallifreyan@protonmail.ch>\n"
 "Language: ru\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 3.4.2\n"
+"X-Generator: Lokalize 24.05.0\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 msgid "translation-credits"
 msgstr "volkov <volkovissocool@gmail.com>"
```

Comment:
Add yourself if you want to be credited

Reply:
do I add another msgid/msgstr or do i append myself to the existing msgstr?

Reply:
I don't remember, look at the German localization file.